environment.yml

```diff
@@ -2,13 +2,13 @@ name: navo-env
 channels:
   - conda-forge
 dependencies:
-  - python=3.9
-  - numpy>=1.24
-  - astropy>=5.2
-  - matplotlib>=3.2
-  - jupyterlab>=3.1
-  - pyvo>=1.4
-  - scipy>=1.0
+  - python=3.11
+  - numpy>=1.26
+  - astropy>=6.0
+  - matplotlib>=3.8
+  - jupyterlab>=4.0
```

Comment:
Heads up: There was an incompatibility with the md notebooks with jupyter-server 2+, so we need to keep an eye on it that none of these brings in that package as a dependency (or if they do, they pick up the correct version)